packages/eas-cli/src/commands/config.ts

```diff
@@ -17,7 +17,7 @@ export default class Config extends EasCommand {
 
   static override flags = {
     platform: Flags.enum({ char: 'p', options: ['android', 'ios'] }),
-    profile: Flags.string(),
+    profile: Flags.string({ char: 'e' }),
```

Comment:
Can we add descriptions to these in this PR or a follow-up PR? It's the only command out of the ones touched that doesn't have flag descriptions.